go.mod

```diff
@@ -11,6 +11,7 @@ require (
 	github.com/emirpasic/gods v1.9.0 // indirect
 	github.com/flynn/go-shlex v0.0.0-20150515145356-3f9db97f8568 // indirect
 	github.com/gaia-pipeline/flag v1.7.4-pre
+	github.com/gaia-pipeline/gosdk v0.0.0-20180909192508-cc9f89055777
```

Comment:
go mod automatically adds them when you test a pipeline build because the source code from the pipeline is in the tmp folder

Reply:
interesting, I thought I made it ignore that folder. :D But apparently not. :)